scrapers/SexLikeReal.yml

```
@@ -7,15 +7,21 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //h1/text()
+      Title: &titleDef
+        selector: //div[@id="tabs-photos"]//a/img/@alt
+        postProcess:
+            - replace:
+                - regex:  " VR porn$"
+                  with: ""
       Date: //time/@datetime
-      Details: //div[@id="modalSceneAbout"]//comment()[contains(.,'description')]/following-sibling::div
+      Details: &detailsSel
+        selector: //div[@data-qa="scene-about-tab-text"]//text()
+        concat: " "
```

```suggestion
      Details: //div[@data-qa="scene-about-tab-text"]
```
Should be enough. Stash gets the innertext of elements when needed